### PR TITLE
Default Environment Variables for Codespaces

### PR DESCRIPTION
**Pull request by geramirez**
### Why:

Closes [issue link]

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: content/codespaces/developing-in-codespaces/default-environment-variables.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+| `CODESPACE_NAME` | Name of the codespace For example:`monalisa-github-hello-world-2f2fsdf2e` |
+| `CODESPACES` | Always `true` while in a codespace |
+| `GITHUB_TOKEN` | undefined |
+| `GIT_COMMITTER_NAME` | undefined |
+| `GIT_COMMITTER_EMAIL` | undefined |
+| `GITHUB_SERVER_URL`| Returns the URL of the {% data variables.product.product_name %} server. For example: `https://{% data variables.product.product_url %}`. |
+| `GITHUB_API_URL` | Returns the API URL. For example: `{% data variables.product.api_url_code %}`. |
+| `GITHUB_GRAPHQL_URL` | Returns the GraphQL API URL. For example: `{% data variables.product.graphql_url_code %}`. |
+| `GITHUB_REPOSITORY` | The owner and repository name. For example, `octocat/Hello-World`. |
+| `INTERNAL_VSCS_TARGET_URL` | undefined |
```
Comment (joshaber) [inline content/codespaces/developing-in-codespaces/default-environment-variables.md:32]: We don't want to document this one, people shouldn't consider this public API.

File: content/codespaces/developing-in-codespaces/default-environment-variables.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+| `GITHUB_CODESPACE_TOKEN` | undefined |
```
Comment (joshaber) [inline content/codespaces/developing-in-codespaces/default-environment-variables.md:33]: Same as above, this isn't public API.

File: content/codespaces/developing-in-codespaces/default-environment-variables.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## List of default environment variables
+
+| Environment variable | Description |
+| ---------------------|------------ |
+| `CODESPACE_NAME` | Name of the codespace For example:`monalisa-github-hello-world-2f2fsdf2e` |
+| `CODESPACES` | Always `true` while in a codespace |
+| `GITHUB_TOKEN` | undefined |
+| `GIT_COMMITTER_NAME` | undefined |
+| `GIT_COMMITTER_EMAIL` | undefined |
```
Comment (joshaber) [inline content/codespaces/developing-in-codespaces/default-environment-variables.md:27]: For these I think we should either give them descriptions or leave them off this list.

**Review by joshaber — COMMENTED** [review]
Thanks for writing this up!

File: content/codespaces/developing-in-codespaces/default-environment-variables.md (hunk 3 above)
Comment (geramirez) [reply content/codespaces/developing-in-codespaces/default-environment-variables.md:27]: yep, this was just a draft. I'll fill these out.

File: content/codespaces/developing-in-codespaces/default-environment-variables.md (hunk 2 above)
Comment (geramirez) [reply content/codespaces/developing-in-codespaces/default-environment-variables.md:33]: got it.

**Pull request closed without merging**